docs/reference/workflow-reference.md

```diff
@@ -1,3 +1,491 @@
+| ------------- | ------------------------------------- |
+| `name`        | **str**<br/>The workflow name.        |
+| `description` | **str**<br/>The workflow description. |
+
+<h5> Example </h5>
+
+```python
+workflow.update_name(
+    name="Sentinel-2 with tiling",
+    description="Implements tiling on free Sentinel-2 data",
+)
+```
+
+### delete()
+
+The `delete()` function allows you to delete a workflow.
+
+```python
+delete()
+```
+
+<h5> Example </h5>
+
+```python
+workflow.delete()
+```
+
+## Workflow tasks
+
+Workflow tasks are blocks that are added to a workflow. A workflow task uses a specific block version that specifies its input JSON parameters and what blocks can be added before and after it.
+
+### workflow_tasks
+
+The `workflow_tasks` attribute returns a list of workflow tasks in a workflow.
+
+The returned format is `dict[str, str]`.
+
+<h5> Example </h5>
+
+```python
+workflow.workflow_tasks
+```
+
+### get_workflow_tasks()
+
+The `get_workflow_tasks` function returns a list of workflow tasks in a workflow.
+
+```python
+get_workflow_tasks(basic)
+```
+
+The returned format is `Union[list, dict]`.
+
+<h5> Arguments </h5>
+
+| Argument | Overview                                                                                                                                                                                                                     |
+| -------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `basic`  | **bool**<br/>Determines how to return a list of workflow tasks:</br/><ul><li>`True`: return only simplified task names and block versions.</li><li>`False`: return the full response.</li></ul>The default value is `False`. |
+
+<h5> Example </h5>
+
+```python
+workflow.get_workflow_tasks(basic=True)
+```
+
+### add_workflow_tasks()
+
+The `add_workflow_tasks()` function allows you to add workflow tasks to a workflow. Include the full sequence of selected workflow tasks as the function overwrites existing workflow tasks.
+
+```python
+add_workflow_tasks(input_tasks)
+```
+
+<h5> Arguments </h5>
+
+| Argument      | Overview                                                                                                                                                                                                            |
+| ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `input_tasks` | **Union[list[str], list[dict]] / required**<br/>The workflow tasks. To specify a specific version of a block, use block IDs. To specify the most recent version of a block, use block names or block display names. |
```

Comment:
Let's move type to a separate column. "Required" is not needed, since it's indicated in the type

Reply:
In which case a task is provided here as a dict?